services/src/main/resources/org/keycloak/protocol/oidc/endpoints/login-status-iframe.ftl

```diff
@@ -49,7 +52,7 @@
             preventAdditionalRequests = true;
           }
 
-          const url = new URL(`${location.origin}${location.pathname}/init`);
+          const url = new URL(window.location.origin + window.location.pathname + "/init");
```

Comment:
Let's not change this code

Reply:
I've changed this just because freemarker has issues with the `$` notation.

Isn't this effectively the same as before?

Reply:
Thanks, updated